hugr-core/tests/snapshots/model__roundtrip_call.snap

```diff
@@ -4,6 +4,10 @@ expression: "roundtrip(include_str!(\"../../hugr-model/tests/fixtures/model-call
 ---
 (hugr 0)
 
+(import prelude.json)
+
+(import arithmetic.int.types.int)
```

Comment:
i was hoping the import would make the subsequent uses more compact - is that not yet the case?

Reply:
At this point it makes subsequent uses more compact in the binary format, since every named reference is replaced with a reference by node index referring to the import node. It doesn't yet allow to "import as". That'd definitely be useful since `arithmetic.int.types.int` is a bit unwieldy. Since that feature can be added independently on top, I propose to do it in a later PR.

Reply:
sounds good yep 👍